client/client.go

```diff
@@ -376,8 +384,8 @@ func (c *client) scheduleUpdateTokenConnection() {
 }
 
 // GetClusterID returns the ClusterID.
-func (c *client) GetClusterID(ctx context.Context) uint64 {
-	return c.svcDiscovery.GetClusterID(ctx)
+func (c *client) GetClusterID(context.Context) uint64 {
```

Comment:
Do we need the context?

Reply:
No need, but it's defined in the Client interfact. I can't change it.

type Client interface {
	// GetClusterID gets the cluster ID from PD.
	GetClusterID(ctx context.Context) uint64